src/firefly/js/core/MasterSaga.js

```diff
@@ -23,13 +23,23 @@ export function dispatchAddSaga(saga, params={}) {
     flux.process({ type: ADD_SAGA, payload: { saga,params}});
 }
 
+/**
+ * Action watcher callback.
+ * @callback actionWatcherCallback
+ * @param {Action} action - the triggering action
+ * @param {function} cancelSelf - a function to cancel this watcher
+ * @param {object} params – params passed through dispatchAddActionWatcher or (if not undefined) the last returned value from the watcher callback
```

Comment:
`params – params`
I think the '-' there is actually an em-dash or long-dash.  It does not show up in my 'US-ASCI' setting.
I am not sure if it'll show up in jsdoc correctly.